integration/src/test/scala/org/knora/webapi/responders/v2/OntologyResponderV2Spec.scala

```diff
@@ -156,6 +161,28 @@ class OntologyResponderV2Spec extends CoreSpec with ImplicitSender {
       )
     }
 
+    "invalidate cached project information when adding an ontology to a project" in {
```

Comment:
`invalidate cached project information when adding an ontology to a project` looks like better shorter PR title ;)